spec/policies/division_policy_spec.rb

```diff
@@ -10,27 +10,41 @@
   context 'being a member of a division' do
     let(:user) { create(:user, :member, division: division) }
 
-    permit_actions [:index, :show]
+    it 'can not index' do
+      expect(described_class.new(user, Division).index?).to be_falsey
+    end
+    permit_actions [:show]
     forbid_actions [:create, :edit, :update, :destroy]
   end
 
   context 'being an admin of a division' do
     let(:user) { create(:user, :admin, division: division) }
 
-    permit_all
+    it 'can index' do
+      expect(described_class.new(user, Division).index?).to be_truthy
+    end
+    permit_actions [:show, :edit, :update]
+    forbid_actions [:create, :destroy]
   end
 
   context 'being a member of a parent division' do
     let(:user) { create(:user, :member, division: parent_division) }
 
-    permit_actions [:index, :show]
+    permit_actions [:show]
```

Comment:
I'm only testing the index permission for the two 'division' contexts.  Since the parent/child relations doesn't impact.  Would you want duplicate tests included for the other permutations anyways?